style.css

```diff
@@ -0,0 +1,199 @@
+body {
+  width: 375px;
+  height: 5147px;
+  background: #f7f7f9;
+}
+
+.header-bg {
+  position: relative;
+  height: 718px;
+  width: 375px;
+  left: 0;
+  right: 0;
+  border-radius: 0 0 0 75px;
+  background: #fff;
+}
+
+#appBar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  top: 0;
+  left: 0;
+}
+
+#toolsBar {
+  position: absolute;
+  top: 44px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  align-content: center;
+  justify-content: center;
+}
+
+.logoPlaceHolder {
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 15px;
+  width: 100px;
+  height: 15px;
+  top: 6px;
+  left: 12px;
+  justify-items: center;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#Hamburger {
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: center;
+  padding: 4px;
+  width: 30px;
+  height: 30px;
+  right: 24px;
+  top: 6px;
+  border-radius: 8px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 10px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  top: 92px;
+  background-image: url(../Portfolio/asset/images/headershape.svg);
+  background-repeat: no-repeat;
+  background-position-y: 15px;
+}
```

Comment:
- [ ] Kindly make sure the background image is visible on the webpage because at this moment we can be able to see the background image on the website.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/219647545-0cd9176c-bf70-42ba-876f-d9a9a1acdbdc.png)

   - ### FIGMA DESIGN
   
   ![image](https://user-images.githubusercontent.com/87197412/219647672-9fcc983f-2bd0-4941-9013-2dfc30c9ba94.png)